localization/ndt_scan_matcher/src/map_update_module.cpp

```diff
@@ -146,12 +146,25 @@ bool MapUpdateModule::update_ndt(const geometry_msgs::msg::Point & position, Ndt
     request,
     [](rclcpp::Client<autoware_map_msgs::srv::GetDifferentialPointCloudMap>::SharedFuture) {})};
 
+  // Wait for maximum 10 milliseconds
+  std::chrono::milliseconds timeout(10);
```

Comment:
@anhnv3991 
Is there a reason for the constraint of 10ms?

Example:
・If driving at a speed of 100km/h, you will move y [m] in x [ms]. If you wait longer than this, there is a risk of moving out of the loaded map area.
・This process normally takes less than 1ms. So 10ms seems abnormal.

Reply:
@YamatoAndo If I understand the async_send_request correctly, wait_for is similar to std future wait_for. That means it waits for a maximum period. So if a response is received before that, it would return immediately with a ready status. If timeout, it would return a timeout status.

Reply:
@anhnv3991 I measured the time it takes for wait_for in my environment, it seems to take about 0 to 300ms depending on the amount of the map being loaded. 
Since there is time required to load the map from the SSD, I think it is impossible to finish within 10ms.

Reply:
@YamatoAndo Can you share your data with 300ms loading time? I want to take a look at them.
300ms loading time sounds too much for me. Loading data from SSD should not take that much, unless your PCD are super dense.

Reply:
@YamatoAndo OK. I will send you via TIER IV slack